### PR TITLE
Add gset support

### PR DESCRIPTION
**Pull request by russelldb**
Rebase of bet365 PR on develop-2.2. Eyes please if you would Mr. @binarytemple-bet365, sir

### CODE REVIEW

**Comment by binarytemple-external** [comment]
LGTM

**Comment by binarytemple-bet365** [comment]
LGTM

**Comment by davidx** [comment]
hi! thumbot likely got confused persisting a head ref name with slashes, will check the logs and make a fix. thx!

**Comment by davidx** [comment]
thumbot retry

File: src/yz_extractor.erl [hunk 1]
```diff
@@ -32,6 +32,7 @@
                       {"application/json",yz_json_extractor},
                       {"application/riak_counter", yz_dt_extractor},
                       {"application/riak_hll", yz_dt_extractor},
+                      {"application/riak_gset", yz_dt_extractor},
```
Comment (binarytemple-external) [inline src/yz_extractor.erl:34]: @russelldb this need to be alphabetically ordered. @lukebakken added a test to ensure they were. If they aren't properly ordered the extraction won't work.

**Comment by binarytemple-external** [comment]
yz_extractor - @russelldb this need to be alphabetically ordered. @lukebakken added a test to ensure they were. If they aren't properly ordered the extraction won't work.

**Comment by russelldb** [comment]
Closing in favour of https://github.com/basho/yokozuna/pull/701

**Comment by rzezeski** [comment]
> This one catches everyone !!!!! 👍

Caught me once or twice and I wrote the damn thing.

**Pull request closed without merging**